docs/self-hosted-appcircle/configure-server/_ufw-configuration.mdx

```diff
@@ -0,0 +1,20 @@
+If you are using `UFW (Uncomplicated Firewall)`, you need to open the 80 and 443 ports for the Appcircle server.
+
+Check if the `ufw` is active.
+
+```bash
+sudo ufw status
+```
+
+If you see `Status: active` as the output, you should allow TCP 80 and TCP 443 ports for Appcircle server to accept connections.
+
+```bash
+sudo ufw allow 80 && \
+sudo ufw allow 443
```

Comment:
Instead of opening both ports, it might be more secure to open ports conditionally. For instance; "if you're using HTTPS bla bla bla ... if you're using HTTP bla bla bla"

Reply:
See [here](https://github.com/appcircleio/appcircle-docusaurus/pull/524#discussion_r1681466911).